lib/mix/tasks/docs.ex

```diff
@@ -140,6 +140,14 @@ defmodule Mix.Tasks.Docs do
     end
   end
 
+  defp fetch_version!(config) do
+    Keyword.fetch!(config, :version)
```

Comment:
I.e. we shouldn't rescue exceptions. I believe when @ericmj mentioned `Keyword.fetch!` he was not expecting a custom error message.